spec/support/helpers.rb

```diff
@@ -41,7 +41,7 @@ def last_command
     end
 
     def out
-      last_command.stdboth
+      last_command.stdout
```

Comment:
This is the key change in this PR. The rest is fixing the tests this change breaks.